operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -8,14 +8,11 @@ import (
 	"fmt"
 
 	"github.com/Jeffail/gabs"
+	"github.com/datastax/cass-operator/operator/pkg/serverconfig"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
-
-	"github.com/datastax/cass-operator/operator/pkg/images"
-	"github.com/datastax/cass-operator/operator/pkg/serverconfig"
-	"github.com/datastax/cass-operator/operator/pkg/utils"
```

Comment:
I was confused at first - but the important thing is to get as few dependencies from this v1beta1 package as possible right? That's something I noticed some time ago, so I agree with this 👍